backend/database/hwSetDB.py

```diff
@@ -1,10 +1,55 @@
-import database.database_methods as db
+import database as db
+from models.HWSet import HWSet
+
+def createHWSetCollection(collectionName):
+    client = db.get_database()
+    projDb = client.SWELabProjectHardwareSets
+    collection = projDb[collectionName]
+    client.close()
+
+def addHWSet(projectId, name, capacity):
+   client = db.get_database()
+   projDb = client.SWELabProjectHardwareSets
+   if projectId not in projDb.list_collection_names():
+      print("Project does not exist")
+      return
+   collection = projDb[projectId]
+   HWSetDoc = HWSet(name, capacity)
+   collection.insert_one(HWSetDoc.to_dict())
+   client.close()
+   
+def getHWSet(projectId, name):
+   client = db.get_database()
+   projDb = client.SWELabProjectHardwareSets
+   if projectId not in projDb.list_collection_names():
+      print("Project does not exist")
+      return
+   collection = projDb[projectId]
```

Comment:
Should query collection of HWSets by HWSet name directly, rather than querying by projectId.